roles/validate_inventory/tasks/vms.yml

```diff
@@ -0,0 +1,82 @@
+- name: Get all KVM Nodes
+  vars:
+    kvm_node_names: []
+  set_fact:
+    kvm_node_names: "{{ kvm_node_names + [item] }}"
+  when: hostvars[item]['vendor'] == 'KVM'
```

Comment:
The new role `process_kvm_nodes` uses case-insensitive conditional checks for `vendor`. Using the same style of checks would make the playbooks more consistent. Personally, I think that the case-insensitive ones are the way to go.
```yaml
when: hostvars[item]['vendor'] | lower == 'kvm'
```

Reply:
I agree I can add a commit and make all the vendor checks case insensitive. I don't think the ones in boot_iso or boot_disk are. What do you think?

Reply:
Sounds good